apps/hubble-stats/app/layout.tsx

```diff
@@ -97,8 +95,9 @@ export default function RootLayout({
             logoLink={WEBB_MKT_URL}
             footer={footer}
           />
-          <main className="flex-1 overflow-y-auto overflow-scroll">
-            <div className="max-w-[1000px] mx-auto">
+
+          <main className="flex-1 overflow-y-auto">
+            <div className="max-w-[1240px] mx-auto">
```

Comment:
@devpavan04 the new size of the main content looks not good on smaller desktop screens. On 1440px, the website looks like this. Also, cc @monaiuu if she wants to add more comments or suggestions on this one

<img width="1440" alt="Screenshot 2023-07-15 at 15 18 39" src="https://github.com/webb-tools/webb-dapp/assets/69841784/3fd0bddd-44ef-4442-8ad5-ffb1964adffb">

Reply:
I have added the responsive breakpoints views for how components should behave across various screensizes. In general, instead of using fixed-width, we should aim for components that 'fill width' and adjust dynamically based on the available space.

Figma Link: https://www.figma.com/file/KWLQWPgzbMMIj0p5dylcgB/%F0%9F%93%88-Shielded-Pool-Stats?type=design&node-id=1298%3A203096&mode=design&t=UsBv5Vq1qhuXY1CM-1